CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_3x3_s8.c

```diff
@@ -49,71 +49,74 @@
    *  Refer prototype header file for details.
    *
    */
-arm_status arm_depthwise_conv_3x3_s8(const int8_t *input,
-                                     const int32_t input_x,
-                                     const int32_t input_y,
-                                     const int32_t input_ch,
-                                     const int8_t *kernel,
-                                     const int32_t output_ch,
-                                     const int32_t pad_x,
-                                     const int32_t pad_y,
-                                     const int32_t stride_x,
-                                     const int32_t stride_y,
+
+arm_status arm_depthwise_conv_3x3_s8(const cmsis_nn_context *ctx,
+                                     const cmsis_nn_dw_conv_params *dw_conv_params,
+                                     const cmsis_nn_per_channel_quant_params *quant_params,
+                                     const cmsis_nn_dims *input_dims,
+                                     const q7_t *input,
+                                     const cmsis_nn_dims *filter_dims,
+                                     const q7_t *kernel,
+                                     const cmsis_nn_dims *bias_dims,
                                      const int32_t *bias,
-                                     int8_t *output,
```

Comment:
output_data?

Reply:
for the _data comments, I think it is ok without the suffix as it is still clear that it is input/output/kernel data.